### ISSUE TITLE
aria-expanded is always true: its value should be passed as boolean not as string

### ISSUE THREAD

**Issue by afercia** [draft-js-plugins/draft-js-plugins#1072]
At the company I work for we've implemented our first draft.js thingy with the mentions plugin, and it's awesome to see it in place in our apps.

However, as an accessibility specialist, I was wondering why the `aria-expanded` HTML attribute is always set to `true` regardless if the "dropdown" is open or closed.

Turns out `draft.js` has changed something since version `0.10.2` and now uses a pattern to make sure `aria-expanded` is only set when there's a `role="combobox"`. That's perfectly fine, but the actual pattern is the following one:

`var ariaExpanded = ariaRole === 'combobox' ? !!this.props.ariaExpanded : null;`

Please notice the double `!!`. It happens that the mentions plugin correctly switches the value on `openDropdown` and `closeDropdown` but then it passes the values as strings `'false'` and `'true'`...

Now, the double `!!` in draft.js casts any non-empty string to boolean `true`... that's the reason why the HTML actual set value is always `true`.

I guess this applies to all the plugins that use `aria-expanded` and pass strings, as far as I see also the Emoji plugin and maybe other ones?

**Comment by juliankrispel** [comment]
awesome! I'll get on it if you don't first.

Also if you feel like opening a pr yourself but don't know where to start, feel free to get in touch on the draft-js slack channel

**Comment by afercia** [comment]
@juliankrispel I have a local branch ready but I'm a bit busy in the next few hours, please do feel free to go head if you have a chance.

**Comment by juliankrispel** [comment]
@afercia I wouldn't be ready until later this week - have loads of other stuff on :D
